Sources/SnowManLogicSpecs/GameSpec.swift

```diff
@@ -0,0 +1,50 @@
+import SnowManLogic
+import Swiftest
+
+class GameSpec: Swiftest.Spec {
+	let spec = describe("game logic") {
+		var guessManager: GuessManager!
+		var word: String!
+		var io: MockIO!
+		var view: View!
+		var game: Game!
+
+		before() {
+			guessManager =  GuessManager(totalIncorrectGuessesAllowed: 5)
+			word = "apple"
+			io = MockIO()
+			view = View(io: io)
+			game = Game(word: word, guessManager: guessManager, view: view)
+		}
+
+		describe("#gameIsOver") {
```

Comment:
I changed this to `isGameOver`. The only reason I didn't change this one was because it felt weirder than `gameIsOver`